### PR TITLE
FEAT: Daphne server for backend microservices

### PR DESCRIPTION
**Pull request by ychaaibi**
Set Daphne.ASGI for backend microservices in the developement and deployment, automate re developement, deploymenet of microservices and reseting data-base in makefile and fix nginx config file.

### **Devops :**

**Deploy microservices as containers to Docker :**

- **_Conf_**

   - _**FIX:**_
      - #101

 

- **_Deployment_**

  -  #102

### CODE REVIEW

File: services/authentication/Dockerfile [hunk 1]
```diff
@@ -6,8 +6,11 @@ RUN         apt update -y && \
 
 RUN         mkdir -p /microservice/authentication/
 
+
 COPY        . /microservice/authentication/
 
+RUN         pip install --upgrade pip
```
Comment (zwzone) [inline services/authentication/Dockerfile:12]: Good job with this one 👍🏼

File: services/authentication/Dockerfile.dev [hunk 2]
```diff
@@ -6,6 +6,12 @@ RUN         apt update -y && \
 
 RUN         mkdir -p /microservice/authentication/
 
+COPY        ./requirements.txt /microservice/player/
+
+RUN         pip install --upgrade pip
```
Comment (zwzone) [inline services/authentication/Dockerfile.dev:11]: 👍🏼

File: services/player/Dockerfile [hunk 3]
```diff
@@ -8,6 +8,8 @@ RUN         mkdir -p /microservice/player/
 
 COPY        . /microservice/player/
 
+RUN         pip install --upgrade pip
```
Comment (zwzone) [inline services/player/Dockerfile:11]: 👍🏼

File: services/player/Dockerfile.dev [hunk 4]
```diff
@@ -4,8 +4,15 @@ RUN         apt update -y && \
             apt upgrade -y && \
 			apt install curl vim nano lsof -y
 
+
 RUN         mkdir -p /microservice/player/
 
+COPY        ./requirements.txt /microservice/player/
+
+RUN         pip install --upgrade pip
```
Comment (zwzone) [inline services/player/Dockerfile.dev:12]: 👍🏼

**Review by tnaceur — APPROVED** [review]

**Review by zwzone — APPROVED** [review]
Amazing and clean

**Pull request merged**